ql/src/test/queries/clientpositive/mytest.q

```diff
@@ -0,0 +1,85 @@
+add jar /home/msydoron/eclipse-workspace/JethroDataJDBCDriver/target/jethro-jdbc-3.6-standalone.jar;
+CREATE EXTERNAL TABLE ext_mytable1 (x1 INT, y1 DOUBLE)
+STORED BY
+'org.apache.hive.storage.jdbc.JdbcStorageHandler'
+TBLPROPERTIES ( "hive.sql.database.type" = "JethroData",
+                "hive.sql.jdbc.driver" = "com.jethrodata.JethroDriver",
+                "hive.sql.jdbc.url" = "jdbc:JethroData://10.0.0.221:9111/demo3",
+                "hive.sql.dbcp.username" = "jethro",
+                "hive.sql.dbcp.password" = "jethro", 
+                "hive.sql.table" = "mytable1",
+                "hive.sql.dbcp.maxActive" = "1");
+                
+CREATE EXTERNAL TABLE ext_mytable2 (x2 INT, y2 DOUBLE)
+STORED BY
+'org.apache.hive.storage.jdbc.JdbcStorageHandler'
+TBLPROPERTIES ( "hive.sql.database.type" = "JethroData",
+                "hive.sql.jdbc.driver" = "com.jethrodata.JethroDriver",
+                "hive.sql.jdbc.url" = "jdbc:JethroData://10.0.0.221:9111/demo3",
+                "hive.sql.dbcp.username" = "jethro",
+                "hive.sql.dbcp.password" = "jethro", 
+                "hive.sql.table" = "mytable2",
+                "hive.sql.dbcp.maxActive" = "1");                
+
+SELECT ext_mytable1.x1, ext_mytable1.y1, ext_mytable2.x2
```

Comment:
Can you add ```explain``` plans for all these queries? That way we can see what is being pushed to underlying JDBC connection and what not, and identify regressions.

For instance, for query ```SELECT a FROM t;```, you can add ```EXPLAIN SELECT a FROM t;``` too.